src/common/CoreMediaIO/DeviceAbstractionLayer/Devices/Sample/PlugIn/CMIO_DP_Sample_Stream.cpp

```diff
@@ -135,6 +147,38 @@ namespace CMIO { namespace DP { namespace Sample
 		mSyncClock(true)
 	{
 	}
+
+DPA::Sample::FrameType getFrameType()
+{
+    cout << "[Debug] REE: Called getFrameType() in CMIO_DP_Sample_Stream.cpp" << endl;
+    obs_video_info ovi;
+    obs_get_video_info(&ovi);
+    stringstream stream;
+    stream << ovi.output_width << "x" << ovi.output_height;
```

Comment:
I'm not at a computer to verify 100% but isn't this run inside the DAL plugin process? It wouldn't be able to read obs data I wouldn't think

Reply:
_(See below, I didn't know you could comment here)_